cvat-core/src/api-implementation.ts

```diff
@@ -34,6 +34,10 @@ import QualitySettings from './quality-settings';
 import { FramesMetaData } from './frames';
 import AnalyticsReport from './analytics-report';
 
+function isPageSize(value: any) {
+    return isInteger(value) || value === 'all';
```

Comment:
It does. It seems a good solution in the current circumstances - otherwise we would probably need to use a loop to fetch all the data, which has bad performance.